php/builder8/Dockerfile.tmpl

```diff
@@ -28,9 +28,9 @@ LABEL org.label-schema.schema-version="1.0" \
       info.humanitarianresponse.php.modules="bcmath bz2 calendar ctype curl dom exif fileinfo fpm ftp gd gettext gmp iconv igbinary imagick imap intl json mcrypt memcached opcache openssl pdo pdo_mysql phar posix redis shmop soap sodium sysvmsg sysvsem sysvshm simplexml sockets xml xmlreader xmlwriter xsl zip zlib" \
       info.humanitarianresponse.php.sapi="cli" \
       info.humanitarianresponse.composer="2.1.5" \
-      info.humanitarianresponse.node="12.18" \
-      info.humanitarianresponse.npm="12.18" \
-      info.humanitarianresponse.yarn="1.7.0" \
+      info.humanitarianresponse.node="16.19.0" \
+      info.humanitarianresponse.npm="8.19.3" \
+      info.humanitarianresponse.yarn="1.22.19" \
```

Comment:
purely curious: you had some console output in the ticket that said npm 9.x but these images have 8.x (which I did mention in the ticket). Is that on purpose?

Either way I'm happy to bump them all the way up to whatever is latest, or stick with 8.x for the time being. We do tend to sit on node/npm versions for a while, so if I had to pick one, I'd go with npm 9.

Reply:
I must have just pasted wrong :-)

Reply:
Ok 👍  I went with 8.19.3